src/main/java/TestFile.java

```diff
@@ -27,7 +27,7 @@ private static void anotherVariationOfIncorrectComparingToItself() {
     }
 
     private static void onMoreVariationOfIncorrectComparingToItself() {
-        int x = 23422;
+        int x = 123123;
```

Comment:
[Checkstyle] WARNING: 'method def' child have incorrect indentation level 8, expected level should be 4.